forks/Microsoft.Health.FellowOakDicom/Serialization/JsonDicomConverter.cs

```diff
@@ -233,7 +233,15 @@ private DicomDataset ReadJsonDataset(ref Utf8JsonReader reader)
 
                     if (dataset.Contains(privateCreatorTag))
                     {
-                        item.Tag.PrivateCreator = new DicomPrivateCreator(dataset.GetSingleValue<string>(privateCreatorTag));
+                        var privateCreatorItem = dataset.GetDicomItem<DicomItem>(privateCreatorTag);
+                        // Based on standard private creator data item should be VM = 1, but there are buggy dcms
+                        // To allow serialization to continue to work instead of throwing checking valid single value before calling GetSingleValue
+                        if (privateCreatorItem is DicomElement element
```

Comment:
We can simplify this to `privateCreatorItem is DicomLongString element` and leave out the VR check.